### PR TITLE
Adding tests for login and register

### PR DESCRIPTION
**Pull request by dhillonks**
Added tests for the following:
- `src/components/SLogin`
- `src/components/Register`

For mocking the requests, I used the already existing `src/testServer.js` module to handle the `login` and `signup` requests.

The tests can be found in:
```
src/components/accounts/__tests__/SLogin.test.js
src/components/accounts/__tests__/Register.test.js
```
image

### CODE REVIEW

**Review by redxzeta — CHANGES_REQUESTED** [review]
I think you could just put fake supabase credentials so it could look like this
```
const SUPABASE_URL = "https://paw-dev.supabase.co";
const KEY = "pawternityhubo";
const supabase = createClient(SUPABASE_URL, KEY);
test("should register sucessfully", async () => {
  render(
    <Provider value={supabase}>
      <Register />
    </Provider>
  );

```

and just change the mock api

can you make the branch based off on `paw-dev` and pr to that branch

File: src/testServer.js [hunk 1]
```diff
@@ -25,11 +26,48 @@ const server = setupServer(
...
 
-beforeAll(() => server.listen());
+beforeAll(() => server.listen({ onUnhandledRequest: "bypass" }));
 afterAll(() => server.close());
-afterEach(() => server.resetHandlers());
+afterEach(() => {
+  server.resetHandlers();
+});
+
+const SUPABASE_URL = "https://vuxohlizzevgcsxlrggf.supabase.co";
```
Comment (redxzeta) [inline src/testServer.js:68]: Was wondering if you could put a dummy supabase url and key

**Comment by dhillonks** [comment]
> I think you could just put fake supabase credentials so it could look like this
> 
> ```
> const SUPABASE_URL = "https://paw-dev.supabase.co";
> const KEY = "pawternityhubo";
> const supabase = createClient(SUPABASE_URL, KEY);
> test("should register sucessfully", async () => {
>   render(
>     <Provider value={supabase}>
>       <Register />
>     </Provider>
>   );
> ```
> 
> and just change the mock api
> 
> can you make the branch based off on `paw-dev` and pr to that branch

Added a supabase client for the test runs, looks like it passed the CI/CD through github now. Also changed the target branch to `paw-dev`. Could you re-review?

**Review by redxzeta — CHANGES_REQUESTED** [review]

File: src/testServer.js (hunk 1 above)
Comment (dhillonks) [reply src/testServer.js]: done!

**Review by redxzeta — APPROVED** [review]

**Pull request merged**